src/platform/plugins/shared/data/public/query/state_sync/types.ts

```diff
@@ -8,8 +8,9 @@
  */
 
 import type { Filter } from '@kbn/es-query';
+import type { RefreshInterval } from '@kbn/data-service-server';
 import type { QueryState } from '../query_state';
-import { RefreshInterval, TimeRange } from '../../../common/types';
+import { TimeRange } from '../../../common/types';
```

Comment:
Might as well update this one too

```suggestion
import type { TimeRange } from '../../../common/types';
```